e2e/testcases/custom_resource_definitions_test.go

```diff
@@ -210,14 +203,6 @@ func TestAddAndRemoveCustomResourceV1(t *testing.T) {
 	addAndRemoveCustomResource(nt, "v1_crds", "anvil-crd.yaml")
 }
 
-func TestAddAndRemoveCustomResourceV1Beta1(t *testing.T) {
-	nt := nomostest.New(t, nomostesting.Reconciliation1)
-	if !nt.SupportV1Beta1CRDAndRBAC() {
-		nt.T.Skip("Kubernetes v1.22 and later do not support the v1beta1 CRD API")
-	}
-	addAndRemoveCustomResource(nt, "v1beta1_crds", "anvil-crd-v1.yaml")
```

Comment:
The `e2e/testdata/customeresources/v1beta1_crds` folder can be deleted as well.

Reply:
done